kerastuner/tuners/hyperband.py

```diff
@@ -51,120 +65,166 @@ class HyperbandOracle(oracle_module.Oracle):
 
     def __init__(self,
                  objective,
-                 max_trials,
+                 max_sweeps,
```

Comment:
Should this argument have a default value? (probably)

Reply:
Changed to `hyperband_iterations` and defaulted to `1`